src/components/base/outline-button/outline-button.css

```diff
@@ -13,23 +19,21 @@
   &:focus {
     @apply bg-neutral-black text-neutral-white;
   }
-
-  &:disabled {
-    @apply cursor-not-allowed text-gray-500 bg-gray-300;
-  }
 }
 
+.btn.primary[aria-disabled='true'] {
+  @apply pointer-events-none cursor-not-allowed text-gray-500 bg-gray-300;
+}
 .btn.secondary {
   @apply bg-gray-300 text-gray-800;
 
   &:hover,
   &:focus {
     @apply bg-neutral-black text-neutral-white;
   }
-
-  &:disabled {
-    @apply cursor-not-allowed text-gray-500 bg-gray-300;
-  }
+}
+.btn.secondary[aria-disabled='true'] {
```

Comment:
Should we move all the disable styles to one place and just separate the selectors with comma. Instead of repeating the same apply three times